service/worker/workerdeployment/replaytester/generate_history.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Consider running this script to generate a new history for TestReplays
+# whenever there's some change to the worker-deployment or worker-deployment-version workflow.
+# To use it, run a local server (any backend) with the dynamic config for worker-versioning 3
+# (system.enableDeploymentVersions=true) enabled and run this script.
+#
+# Note: this requires temporal cli >= 0.12 and sdk >= v1.33.0
+
+deploymentName="foo"
+version="1.0"
+
```

Comment:
make sure that when you run this script, the server you used to generate the worker deployment wf was the old server version (without the changes in this branch)

Reply:
[v1.27.1](https://github.com/temporalio/temporal/releases/tag/v1.27.1) is the latest OSS version.
[v1.27.0-128.5](https://github.com/temporalio/temporal/releases/tag/v1.27.0-128.5) is the latest OSS tag used in cloud (used in [v3.128.5+oss1.27.0-128.5](https://github.com/temporalio/saas-temporal/releases/tag/v3.128.5%2Boss1.27.0-128.5) of saas-temporal).